blocks/embed/embed.js

```diff
@@ -142,6 +142,19 @@ const loadEmbed = (block, grandChilds, link, existingClassList) => {
     block.classList = `block embed embed-${config.match[0]}`;
   } else {
     block.innerHTML = getDefaultEmbed(url);
+    // Pass the video config to the iframe
+    const videoConfig = {
+      autoplay: 'muted',
```

Comment:
Can we try with the value `any`?

with reference to https://videojs.com/guides/options/#autoplay.